src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AzureRefactoringTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.Testing;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+
+namespace Azure.ClientSdk.Analyzers.Tests
+{
+    public class AzureRefactoringTest<TRefactoring> : CSharpCodeRefactoringTest<TRefactoring, XUnitVerifier> where TRefactoring : CodeRefactoringProvider, new()
+    {
+        private static readonly ReferenceAssemblies DefaultReferenceAssemblies =
+            ReferenceAssemblies.Default.AddPackages(ImmutableArray.Create(
+                new PackageIdentity("Microsoft.Bcl.AsyncInterfaces", "1.1.0"),
+                new PackageIdentity("System.Threading.Tasks.Extensions", "4.5.3")));
```

Comment:
Don't you need references to at least netstandard.dll?

Reply:
The base infrastructure figures out the base references, these are the additional ones.

Reply:
Ah -- that's nice! That must be a new feature (or automatically done for refactoring vs analyzers).